content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md

```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.27"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.32"
 ---
```

Comment:
Is this a good place to add description of[ InPlacePodVerticalScalingExclusiveCPUs  and 
InPlacePodVerticalScalingAllocatedStatus](https://github.com/kubernetes/kubernetes/blob/c9092f69fc0c099062dd23cd6ee226bcd52ec790/pkg/features/kube_features.go#L284-L297) ?